packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -647,19 +648,19 @@ export class ConnectionManager implements IConnectionManager {
 
 				const waitStartTime = performance.now();
 				const retryDelayFromError = getRetryDelayFromError(origError);
-				if (retryDelayFromError !== undefined) {
-					// If the error told us to wait, then we wait.
-					this.props.reconnectionDelayHandler(retryDelayFromError, origError);
+				if (retryDelayFromError !== undefined || globalThis.navigator?.onLine !== false) {
```

Comment:
I'd not use globalThis.navigator?.onLine. It's not reliable. Maybe we can use it as add-on to other logic that is more reliable.
We have talked in the past that we will handle differently cases where we got some error from SPO (and thus we know SPO is reachable) vs. where we were not able to reach it. For later, I'd use shorter max delay

Reply:
We already have this logic in place: "We have talked in the past that we will handle differently cases where we got some error from SPO (and thus we know SPO is reachable) vs. where we were not able to reach it".  calculateMaxWaitTime(origError) does that. We are only using "globalThis.navigator?.onLine" in case it is not supplied from error.